### PR TITLE
Feature/queues

### PR DESCRIPTION
**Pull request by tyler27**

### CODE REVIEW

File: script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts [hunk 1]
```diff
@@ -27,7 +26,6 @@ onPacketHandler<OpNPCPacket> {
 
     with(player) {
         cancelAll()
-        faceActor(npc::index)
```
Comment (ultraviolet-jordan) [inline script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts:30]: Can you undo this change?

**Review by ultraviolet-jordan — CHANGES_REQUESTED** [review]

File: game/src/main/kotlin/xlitekt/game/tick/benchmark/BenchmarkParallelClientSynchronizer.kt [hunk 2]
```diff
@@ -48,6 +47,7 @@ class BenchmarkParallelClientSynchronizer : Synchronizer() {
         val playerSyncFirstBlock = measureTime {
             players.parallelStream().forEach {
                 it.invokeAndClearReadPool()
+                it.process()
```
Comment (ultraviolet-jordan) [inline game/src/main/kotlin/xlitekt/game/tick/benchmark/BenchmarkParallelClientSynchronizer.kt:50]: This function needs to be added to all of the synchronizers.

**Review by ultraviolet-jordan — CHANGES_REQUESTED** [review]

File: script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts (hunk 1 above)
Comment (ultraviolet-jordan) [reply script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts:30]: This probably does inside queue (you will have to queue the npc routing and loc routing as well)

Comment (tyler27) [reply script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts:30]: Yeah, we also need to add the queueStrong {} to all of the movement related stuff

File: game/src/main/kotlin/xlitekt/game/tick/benchmark/BenchmarkParallelClientSynchronizer.kt (hunk 2 above)
Comment (tyler27) [reply game/src/main/kotlin/xlitekt/game/tick/benchmark/BenchmarkParallelClientSynchronizer.kt:50]: kk done

**Review by ultraviolet-jordan — APPROVED** [review]

File: script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts (hunk 1 above)
Comment (tyler27) [reply script/src/main/kotlin/script/packet/disassembler/handler/OpNPCHandler.kts:30]: done
